drivers/i2c/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+zephyr_sources_if_kconfig(i2c_imx.c)
```

Comment:
Seems like we should keep this the same as the other lines:

zephyr_source_ifdef(CONFIG_I2C_IMX . i2c_imx.c)